src/build.ts

```diff
@@ -39,7 +39,7 @@ export async function build({ script = 'sync', mock = false } = {}) {
 
   const ssrConfig: UserConfig = {
     esbuild: {
-      target: 'es2018',
+      target: 'es2020',
     },
```

Comment:
We can just remove this section to align with client build (up to user's config, if any)